.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  pull_request:
+
+permissions:
+  packages: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@2b82ce82d56a2a04d2637cd93a637ae1b359c0a7 # v2.2.0
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@4c0219f9ac95b02789c1075625400b2acbff50b1 # v2.9.1
+
+      - name: Login to GHCR
+        if: github.ref == 'refs/heads/main'
+        uses: docker/login-action@465a07811f14bebb1938fbed4728c6a1ff8901fc # v2.2.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@818d4b7b91585d195f67373fd9cb0332e31a7175 # v4.6.0
+        with:
+          images: |
+            ghcr.io/${{ github.repository }}
```

Comment:
ii think there should be an image name? otherwise it would published as organization image only?

Reply:
`${{ github.repository }}` resolves to `renovatebot/renovate-approve-bot` in this case, so the image will automatically be "attached" to this repo. 

You can check that out at https://github.com/anaconda/renovate-approve-bot/commit/d465678033fb435ee2a97ccd07c99e041948d63c, I pushed this exact commit to `main` there to verify that it works.

As far as I'm aware, packages are always on an org/user level on GitHub, we can just link them to a repo, too?

Reply:
you are right, my mistake.